cmake-ide.el

```diff
@@ -470,7 +501,8 @@ flags."
       (compile (cmake-ide--get-compile-command cmake-ide-dir))
     (let ((command (read-from-minibuffer "Compiler command: " compile-command)))
       (compile command)))
-  (cmake-ide--run-rc))
+  (cmake-ide--run-rc)
+  (cmake-ide--set-cmake-dir-for-irony))
```

Comment:
That's the 2nd time these two get called in succession. It'd be bettter to define a new function that calls these two and call it instead in the 2 locations.